src/lib/stitching/convection/stitching.ts

```diff
@@ -0,0 +1,37 @@
+import gql from "test/gql"
+import { GraphQLSchema } from "graphql"
+
+export const consignmentStitchingEnvironment = (
+  localSchema: GraphQLSchema,
+  convectionSchema: GraphQLSchema
+) => ({
+  // The SDL used to declare how to stitch an object
+  extensionSchema: gql`
+    extend type ConsignmentSubmission {
+      artist: Artist
+    }
+  `,
+
+  // Resolvers for the above
+  resolvers: {
+    ConsignmentSubmission: {
+      artist: {
+        fragment: `fragment SubmissionArtist on Submission { artist_id }`,
+        resolve: (parent, args, context, info) => {
+          const id = parent.artist_id
+          return info.mergeInfo.delegateToSchema({
+            schema: localSchema,
+            operation: "query",
+            fieldName: "artist",
+            args: {
+              id,
+            },
+            context,
+            info,
+            transforms: convectionSchema.transforms,
+          })
+        },
+      },
+    },
+  },
+})
```

Comment:
Basically lets each service keep their stitching to themselves. If/when we get to a point where two are trying to stitch resolvers on to the same object, we can deal with it then